Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenContract.swift

```diff
@@ -24,6 +24,7 @@ protocol AboutScreenInteractorInput: AnyObject {
     func fetchBundleInfo()
 }
 
+// sourcery: AutoMockable
 protocol AboutScreenInteractorOutput: AnyObject {
-    func interactor(_: AboutScreenInteractorInput, didFetchBundleInfo bundleInfo: BundleInfo)
+    func interactor(_ interactor: AboutScreenInteractorInput, didFetchBundleInfo bundleInfo: BundleInfo)
```

Comment:
@Dm94st Without local name for variables in protocol AutoMockable generates wrong method signatures. I will consider specifying local name to be best practice further.

Reply:
If I'm not mistaken, `Sourcery` should support `stencil` templates for mock objects. I think we need to use this way instead of default templates to avoid such problems.